inference-engine/thirdparty/clDNN/src/layout_optimizer.cpp

```diff
@@ -1333,7 +1333,7 @@ format layout_optimizer::get_preferred_format(program_node& node) {
         auto& input = node.get_dependency(0);
         if (use_onednn_impls && input.get_output_layout().size.feature[0] == 1 && input.is_in_data_flow() && !input.is_constant()) {
             auto in_layout = input.get_output_layout();
-            if (in_layout.format == format::bfyx && output_layout.size.feature[0] >= 16)
+            if (in_layout.format == format::bfyx && output_layout.size.feature[0] >= 16 && expected != format::bfyx)
```

Comment:
need to check whether the format is planar or not. `byxf` needs to be taken into consideration.